dfx.json

```diff
@@ -488,6 +488,29 @@
 				}
 			}
 		},
+		"xtc_ledger": {
+			"type": "custom",
+			"build": "scripts/build.xtc_ledger.sh",
+			"candid": "target/xtc_ledger.did",
+			"wasm": "target/xtc_ledger.wasm.gz",
+			"specified_id": "aanaa-xaaaa-aaaah-aaeiq-cai",
```

Comment:
This is the sonic dapp canister? https://dashboard.internetcomputer.org/canister/aanaa-xaaaa-aaaah-aaeiq-cai